src/content/content.json

```diff
@@ -42,6 +42,15 @@
         "paragraph_two": "Drexel EWB wants to help the community regain its access and security to clean drinking water. The next step in helping the community is to install covers for the community’s clarifiers. Clarifiers are a component of a water distribution system in which flow is slowed to allow heavy particulates to sink to the bottom, leaving the top more potable and less turbid. Currently, in San Luis, these clarifiers are exposed to the open air and therefore easily contaminated by volcanic ash, bathing animals, and debris that blows in. Through a clarifier cover installation, the water in this phase of the system will be protected from natural forces that could cause contamination. Additionally, Drexel EWB is currently exploring the idea of replacing the pipes which carry the water from the community’s natural spring sources to the clarifier and storage tanks. Somewhere along the route, there are issues of flow inconsistencies and leakages which compromise the reliability of the system. Our next trip will be our first implementation trip and we plan to install the clarifier covers and collect more data to determine if replacing these pipes is necessary. Due to COVID-19, the trip is currently being planned for March 2021, as the pandemic has pushed our September 2020 plan."
       }
+    },
+    "sanctuary_farm": {
+      "header": "Sanctuary Farm - United States",
+      "summary": {
+        "paragraph_one": "Sanctuary Farm is a nonprofit urban garden located in North Philadelphia which the Domestic Design Team has partnered with since 2017. Over the past 5 years, the team has completed many projects for this organization including irrigation system design and installation, solar panel installation and connection, as well as rainwater catchment system installation. Most recently the team set up the fans, motors, and temperature sensors in a new high tunnel and connected them so that they are powered entirely by four solar panels on site. With this high tunnel, the garden will be able to produce more food and provide greater access to fruits and vegetables for the surrounding community throughout the year. It has been exciting to help facilitate this garden’s growth and we hope to continue the partnership for many years to come.        ",
+        "paragraph_two": ""
```

Comment:
"paragraph_two" section can be deleted. We do not need to include it in the content file if we are not going to use it.